cbw_api_toolbox/cbw_api.py

```diff
@@ -167,3 +168,45 @@ def update_remote_access(self, remote_access_id, info):
 
         logging.error("Error update remote access")
         return False
+
+    def export_remotes_accesses_files_xlsx(self,
```

Comment:
Same as the other PR #35, this is in the wrong file.

Reply:
I'm waiting for my PR # 35 to be valid for not doing double work